app/_hub/kong-inc/mocking/index.md

```diff
@@ -537,12 +556,27 @@ vary: Origin
 
 ### Test the mock response
 
-Test from within the Dev Portal spec Try it Out:
+Test the mock response from within the Dev Portal spec using the Try it out feature:
+
+1. From the Dev Portal home page, click the **Stock API** Service tile.
+
+   ![Dev Portal Services](/assets/images/docs/dev-portal/stock-service.png)
+
+2. Click the **GET /stock/historical** method and the **Try it out** button.
+
+3. Enter the ticker sign **AAPL** in the **tickers** box and click **Execute**.
 
+   The Server response is displayed.
+
+   ![Try it out](/assets/images/docs/dev-portal/response_placeholder.png)
```

Comment:
I'm not sure why but I keep getting TypeError: Failed to fetch even though I'm trying to follow the video and user guide

Reply:
Thanks @murillopaula for figuring this out with me today. Added to troubleshooting.

Reply:
Update: Shane requested removing troubleshooting and changing the example spec instead.